eng/pipelines/common/upload-unsigned-artifacts-step.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
+  name: ''
+
+steps:
+- task: CopyFiles@2
+  displayName: Prepare job-specific intermediate unsigned artifacts subdirectory
+  inputs:
+    SourceFolder: '$(Build.SourcesDirectory)/artifacts/packages/$(_BuildConfig)'
+    Contents: |
+      Shipping/**/*
+      NonShipping/**/*
+    TargetFolder: '$(Build.StagingDirectory)/UnsignedArtifacts/${{ parameters.name }}'
+    CleanTargetFolder: true
+  condition: and(succeeded(), eq(variables._BuildConfig, 'Release'))
```

Comment:
Is the _BuildConfig condition in these two jobs necessary? We just import this template when isOfficialBuild == true.

Reply:
Good call, I believe it's just a carryover from core-setup where Release and Debug were always built--the official build served as rolling test CI as well as the official build. Will remove.

Reply:
Then we can remove the condition as the default when no condition is provided is `succeeded()`.